src/utils/response.ts

```diff
@@ -0,0 +1,21 @@
+export const response = {
+  success: <T>(data: T, message?: string) => ({
```

Comment:
Your usage of generics here is cool. I might decimate this when I test what you've done though

Reply:
Its just to take what ever form of data that is been passed to the response